en/docs/assets/css/config-catalog.css

```
@@ -0,0 +1,288 @@
+.mb-config-catalog .mb-config>.config-wrap>code {
+    font-weight: bold;
+    font-size: 105%;
+}
+
+.mb-config-catalog .mb-config .param {
+    display: table-row;
+}
+
+.mb-config-catalog .mb-config .param>.param-name {
+    display: table-cell;
+    vertical-align: top;
```

Stylelint (17.4.0)

[error] Unexpected duplicate "vertical-align" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)